_maps/map_files/Deltastation/DeltaStation2.dmm

```diff
@@ -55424,7 +55424,10 @@
 	req_access_txt = "61"
 	},
 /turf/open/floor/plasteel/vault{
-	dir = 8
+	dir = 5;
+	initial_gas_mix = "n2=100;TEMP=80";
+	tag = "icon-vault (NORTHEAST)";
```

Comment:
Multiple identical tags cause issues with locate since they can't use /ref macro to get unique reference. We have a lot of them since the generate from states/directions command in dm autoappends them.